analyzers/SonarAnalyzer.sln

```diff
@@ -57,7 +57,17 @@ Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "SonarAnalyzer.Shared", "src
 EndProject
 Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "GitHub", "..\.github\GitHub.shproj", "{532AC70E-90CB-48AD-86F4-16767C22314D}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SonarAnalyzer.ShimLayer.CodeGeneration", "src\SonarAnalyzer.ShimLayer.CodeGeneration\SonarAnalyzer.ShimLayer.CodeGeneration.csproj", "{684F0AD7-BBC9-40C8-9E54-D9C2B57560D9}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SonarAnalyzer.ShimLayer.CodeGeneration", "src\SonarAnalyzer.ShimLayer.CodeGeneration\SonarAnalyzer.ShimLayer.CodeGeneration.csproj", "{684F0AD7-BBC9-40C8-9E54-D9C2B57560D9}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ITs", "ITs", "{314A0C27-4768-4F15-B7FA-DC3C33C08180}"
+EndProject
+Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "config", "its\config\config.shproj", "{5D6668F5-61B7-488E-9B8A-958FCEF05E71}"
```

Comment:
Can we make these look nice pleaaase 😃 (Git isn't friendly with case-only file renames. I'm usually creating commit like `xConfig.shproj` and second with `Config.shproj`)
```suggestion
Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "Config", "its\config\Config.shproj", "{5D6668F5-61B7-488E-9B8A-958FCEF05E71}"
```

Reply:
I just renamed the projects to upper-case and re-imported them